firenoc/firenoc-services/src/api/search.js

```
@@ -194,16 +194,12 @@ export const searchApiResponse = async (request, next = {}) => {
   }
-  if (queryObj.hasOwnProperty("ids")) {
-    // console.log(queryObj.ids.split(","));
-    let ids = queryObj.ids.split(",");
-    sqlQuery = `${sqlQuery} FN.uuid IN ( `;
-    for (var i = 0; i < ids.length; i++) {
-      sqlQuery = `${sqlQuery} '${ids[i]}' `;
-      if (i != ids.length - 1) sqlQuery = `${sqlQuery} ,`;
-    }
 
-    if (ids.length > 1) sqlQuery = `${sqlQuery} ) AND`;
+  if (queryObj.hasOwnProperty("ids")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)